### DISCUSSION TITLE
GitHub Actions triggered for branch that doesn't meet the criteria

### DISCUSSION THREAD

**Discussion by iOSDeveloperL** [community/community#27000] [Actions]
<p>I want to do some actions only if there’s a new branch that was created with <code>release/</code> or <code>hotfix/</code> prefix. This is what I came up with:</p>
<pre><code>name: Release Branch created
on:
  create:
    branches:
      - 'release/**'
- 'hotfix/**'
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest
    steps:
      - name: Do stuff
</code></pre>
<p>This doesn’t work as intended. this job is triggerred for random branch names like <code>test/hello</code>. Any idea why?</p>

**Comment by chrispat** [comment, marked as answer]
<p>The <code>create</code> and <code>delete</code> events do not support filtering like <code>push</code> and <code>pull_request</code> do.</p>
